File: Meadow_DotNet_SDK/Project_Templates/templates/MeadowApplicationCoreComputeVBNet/MeadowApp.vb
Imports System.Threading
Imports Meadow
Imports Meadow.Devices
Imports Meadow.Foundation
Imports Meadow.Foundation.Leds
Imports Meadow.Peripherals.Leds

Public Class MeadowApp
    ' Change F7CoreComputeV2 to F7FeatherV2 (or F7FeatherV1) for Feather boards
    Inherits App(Of F7CoreComputeV2)

    Public Overrides Function Run() As Task
        Resolver.Log.Info("Run... (VB.NET)")

        Resolver.Log.Info("Hello, Meadow Core-Compute!")

        Return MyBase.Run()
    End Function

    Public Overrides Function Initialize() As Task
        Resolver.Log.Info("Initialize... (VB.NET)")

        Return MyBase.Run()
    End Function
End Class

File: Meadow_DotNet_SDK/Project_Templates/templates/Meadow.StartKit/StartKit.F7Feather/MeadowApp.cs
using System.Threading.Tasks;
using Meadow;
using Meadow.Devices;
using $safeprojectname$.Core;

namespace $safeprojectname$.F7Feather
{

    public class MeadowApp : App<F7FeatherV2>
    {
        private $safeprojectname$Platform _platform;

        public override async Task Initialize()
        {
            _platform = new $safeprojectname$Platform(Device);
            var c = new MainController();
            await c.Initialize(_platform);
            _ = c.Run();
        }
    }
}